pkg/vm/machine.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	virtioRootFsTag = "/dev/root"
+	virtioRootFsTag = "vroot"
```

Comment:
Not sure I understand this one

Reply:
this is a tag that is given to the rootfs virtiofs and is needed to be used to mount the `/` inside the VM from a virtiofs that is shared from the host. the problem is ubuntu is really really not happy that the tag has `/dev` prefix because now it thinks it's an actual device then fails to mount the filesystem because of checks did not pass.

Hence i had to change the tag. the tag has no special meaning and can be anything (was recommended by CH to use /dev/root) because it's basically and id of which fs to mount.

Changing this to `vroot` or any other unique string should fix the problem